### ISSUE TITLE
为什么提取特征后，再进行normlization 呢？

### ISSUE THREAD

**Issue by ronyuzhang** [deepinsight/insightface#232]
embedding = sklearn.preprocessing.normalize(embedding).flatten()

**Comment by wayen820** [comment]
看看作者论文
